packages/bpk-component-fieldset/src/BpkFieldset.scss

```diff
@@ -29,4 +29,10 @@
     @include bpk-apply-gray-500-to(color);
     @include bpk-text--xs;
   }
+
+  &--invalid {
+    label {
```

Comment:
This isn't a pattern we generally use. Instead could we create the following style:
```
&__label {
  &--invalid {
    color: $bpk-form-validation-color;
  }
}
```

And then apply the className `bpk-fieldset__label--invalid` to the `BpkLabel` element?